joss-paper/paper.md

```diff
@@ -0,0 +1,159 @@
+---
+title: 'PyFstat: a Python package for continuous gravitational-wave data analysis'
+tags:
+  - Python
+  - gravitational waves
+  - continuous waves
+  - pulsars
+  - data analysis
+authors:
+  - name: David Keitel
+    orcid: 0000-0002-2824-626X
+    affiliation: "1"
+  - name: Rodrigo Tenorio
+    orcid: 0000-0002-3582-2587
+    affiliation: "1"
+  - name: Gregory Ashton
+    orcid: 0000-0001-7288-2231
+    affiliation: "2"
+  - name: Reinhard Prix
+    orcid: 0000-0002-3789-6424
+    affiliation: "3, 4"
+affiliations:
+  - name: Departament de Física, Institut d'Aplicacions Computacionals i de Codi Comunitari (IAC3), Universitat de les Illes Balears, and Institut d'Estudis Espacials de Catalunya (IEEC), Crta. Valldemossa km 7.5, E-07122 Palma, Spain
+    index: 1
+  - name: OzGrav, School of Physics & Astronomy, Monash University, Clayton 3800, Victoria, Australia
+    index: 2
+  - name: Max-Planck-Institut für Gravitationsphysik (Albert-Einstein-Institut), D-30167 Hannover, Germany
+    index: 3
+  - name: Leibniz Universität Hannover, D-30167 Hannover, Germany
+    index: 4
+date: 08 January 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Gravitational waves in the sensitivity band of ground-based detectors
+can be emitted by a number of astrophysical sources,
+including not only binary coalescences, but also individual spinning neutron stars.
+The most promising signals from such sources,
+although as of 2020 not yet detected,
+are the long-lasting, quasi-monochromatic 'Continuous Waves' (CWs).
+Many search methods have been developed and applied on
+LIGO [@TheLIGOScientific:2014jea]
+and Virgo [@TheVirgo:2014hva] data.
+See @Prix:2009oha, @Riles:2017evm, and @Sieniawska:2019hmd for reviews of the field.
+
+The `PyFstat` package provides a convenient interface
+to perform a range of CW data analysis tasks.
+It revolves around the $\mathcal{F}$-statistic,
+first introduced by @Jaranowski:1998qm:
+a matched-filter detection statistic for CW signals
+described by a set of frequency evolution parameters
+and maximized over amplitude parameters.
+It has been one of the standard methods for LIGO-Virgo CW searches for two decades.
+`PyFstat` is built on top of established routines
+in `LALSuite` [@lalsuite]
+but enables a flexible approach to designing new search strategies.
```

Comment:
```suggestion
enabling a flexible approach to designing new search strategies.
```

Reply:
I'd like to keep putting emphasis on the contrast here: "yes it reuses existing stuff, BUT it enables something new". If that wasn't clear enough, it could be rewritten a bit more explicitly, though. How about we put back in the "interface" edited out before?
```suggestion
but through its more modern `python` interface
it enables a flexible approach to designing new search strategies.
```

Reply:
Not my cup of tea, but definitely better than before. I'll let you press the commit button.